docker/dev/Dockerfile

```diff
@@ -86,7 +93,7 @@ COPY ./docker/dev/vpp.conf /etc/vpp/vpp.conf
 COPY ./docker/dev/init_hook.sh /usr/bin/
 
 # Build agent
-WORKDIR $GOPATH/src/github.com/ligato/vpp-agent
+WORKDIR /src/ligato/vpp-agent
 
 ARG VERSION
 ARG COMMIT
```

Comment:
Can you also add this lines, so only in case of a change in the dependencies, docker do `RUN go mod download` and use cached layers otherwise? It would help with rebuilding image with only source code changes.
```diff
 ARG COMMIT
 ARG DATE
 
+COPY go.mod go.sum ./
+RUN go mod download
+
 COPY . ./
```